components/Evaluator/AddTagModal.js

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import Button from '../button';
+import Tag from '../tag';
+import { ASSESSMENT_COLOR, COLOR } from '../../constants';
+import { updateApplicantTags } from '../../utility/firebase';
+
+const ModalContainer = styled.div`
+  background: white;
+  border: 1px solid #d2d2d2;
+  box-sizing: border-box;
+  box-shadow: 0px 4px 20px rgba(63, 63, 63, 0.15);
+  border-radius: 5px;
+  padding: 16px 11px;
+  min-width: 226px;
+  position: absolute;
+  bottom: 0;
+`;
+
+const TitleContainer = styled.span`
+  font-family: HK Grotesk;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  color: ${ASSESSMENT_COLOR.DARK_GRAY};
+`;
+
+const ExistingTagsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 9px;
+  margin: 10px 0;
+`;
+
+const NotAddedTagsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 10px 0 20px;
+`;
+
+const TagDropDown = styled.div`
+  cursor: pointer;
+  padding: 4.5px 0 4.5px 5px;
+  margin-left: -5px;
+
+  &:hover {
+    background: ${ASSESSMENT_COLOR.LIGHT_BLUE};
+  }
+`;
+
+function NotAddedTags({ hackerId, allTags, applicantTags = [] }) {
+  const [tags, setTags] = useState([]);
+
+  useEffect(() => {
+    const notAddedTags = [];
+    for (const tag of allTags) {
+      let exists = false;
+      for (const addedTag of applicantTags) {
+        if (tag.text === addedTag.text) exists = true;
+      }
+      if (!exists) notAddedTags.push(tag);
+    }
```

Comment:
Could we do a `.filter()` here instead? Filter on the `allTags` for tags that are not in `applicantTags`?